download_data.sh

```diff
@@ -2,25 +2,26 @@
 
 mkdir -p 'data/raw/'
 
-# Download synapse data - must be approved user:
+# Download synapse data from Data Freeze Version 1.3
 # https://www.synapse.org/#!Synapse:syn4557014
+# Must be an approved Synapse user
 
-# Normalized RNAseq data
+# Normalized RNAseq data (version 3 - modified on 2016-03-24)
+# md5: 02e72c33071307ff6570621480d3c90b
 synapse get syn4976369
 mv EB++AdjustPANCAN_IlluminaHiSeq_RNASeqV2.geneExp.tsv \
 data/raw/pancan_normalized_rnaseq.tsv
 
-# Clinical data
-synapse get syn4983466
-mv clinical_PANCAN_patient_with_followup.tsv data/raw/pancan_clinical.tsv
+# Clinical sample freeze info (downloaded 11 Jan 2017)
+# Google doc located here
+# https://docs.google.com/spreadsheets/d/1Z1H3mXdO_sk9nc0v8df7VNS_XzXiu6vKGJRbK1qYoh4/edit?usp=gmail
```

Comment:
Not sure if this would be useful for you, but you can retrieve the TSV of Google Sheets from a URL like:

```
https://docs.google.com/spreadsheets/d/1Z1H3mXdO_sk9nc0v8df7VNS_XzXiu6vKGJRbK1qYoh4/export?format=tsv&gid=1397969925
```

`gid` specifies which sheet (spreadsheet tab) to export.